sympy/functions/elementary/complexes.py

```diff
@@ -896,18 +1064,32 @@ class periodic_argument(Function):
     logarithm. That is, given a period P, always return a value in
     (-P/2, P/2], by using exp(P*I) == 1.
 
+    Examples
+    ========
+
     >>> from sympy import exp, exp_polar, periodic_argument, unbranched_argument
```

Comment:
You may have to delete unused import `periodic_argument` to pass flake8 test

Reply:
Yeah. I added another example with periodic argument instead.